apps/condo/domains/ticket/schema/TicketShareService.js

```diff
@@ -0,0 +1,55 @@
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const { COUNTRIES, RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
+const { SHARE_TICKET_MESSAGE_TYPE } = require('@condo/domains/notification/constants')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const { Ticket } = require('@condo/domains/ticket/utils/serverSchema')
+
+const TicketShareService = new GQLCustomSchema('TicketShareService', {
+    types: [
+        {
+            access: true,
+            type: 'input TicketShareInput { sender: JSON!,  users: [ID!]!, ticketId: ID! }',
+        },
+        {
+            access: true,
+            type: 'type TicketShareOutput { status: String! }',
+        },
+    ],
+    mutations: [
+        {
+            access: true,
+            schema: 'ticketShare(data: TicketShareInput!): TicketShareOutput',
+            resolver: async (parent, args, context) => {
+                const { data } = args
+                const { users, ticketId, sender } = data
+                const [ticket] = await Ticket.getAll(context, { id: ticketId })
+
+                const lang = COUNTRIES[RUSSIA_COUNTRY].locale
+
+                await Promise.all(users.map( id => sendMessage(context, {
+                    lang,
+                    to: {
+                        user: {
```

Comment:
Maybe, we can send 1 message to all users?

Reply:
No. If I understood correctly, only one user or user ID can be passed, an array is not accepted.